sormas-api/src/main/java/de/symeda/sormas/api/symptoms/SymptomsDto.java

```diff
@@ -481,9 +483,13 @@ public static SymptomsDto build() {
 	private String congenitalHeartDiseaseDetails;
 	@Diseases({Disease.EVD,Disease.LASSA,Disease.YELLOW_FEVER,Disease.DENGUE,Disease.PLAGUE,Disease.UNSPECIFIED_VHF,Disease.UNDEFINED,Disease.OTHER})
 	private SymptomState unexplainedBleeding;
-	@Diseases({Disease.EVD,Disease.LASSA,Disease.NEW_INFLUENCA,Disease.CSM,Disease.CHOLERA,Disease.MEASLES,Disease.YELLOW_FEVER,Disease.DENGUE,Disease.MONKEYPOX,Disease.PLAGUE,Disease.UNSPECIFIED_VHF,Disease.UNDEFINED,Disease.OTHER})
+	@Diseases({Disease.EVD,Disease.LASSA,Disease.NEW_INFLUENCA,Disease.CSM,Disease.CHOLERA,Disease.MEASLES,Disease.YELLOW_FEVER,Disease.DENGUE,Disease.MONKEYPOX,Disease.PLAGUE,Disease.ANTHRAX,Disease.UNSPECIFIED_VHF,Disease.UNDEFINED,Disease.OTHER})
 	@Outbreaks
 	private SymptomState vomiting;
+	@Diseases({Disease.ANTHRAX})
+	private SymptomState convulsion;
+	@Diseases({Disease.ANTHRAX})
+	private SymptomState skinLesion; /* oedema */
```

Comment:
Take this symptom out; I've talked to Bernard and he said that we should only use one of the lesion symptoms, i.e., the one that's already existing